railties/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+*   Allow users to silence the "Rails couldn't infer whether you are using multiple databases..."
+    message using `config.active_record.suppress_multiple_database_warning`.
+
+    *Omri Gabay*
```

Comment:
This should be in Active Record since it's an active record configuration option.